scripts/listChangedFiles.js

```diff
@@ -0,0 +1,36 @@
+// Based on material-ui's prettier scripts
+
+const util = require('util');
+const childProcess = require('child_process');
+
+const execFileAsync = util.promisify(childProcess.execFile);
+
+async function exec(command, args) {
+  const options = {
+    cwd: process.cwd(),
+    env: process.env,
+    stdio: 'pipe',
+    encoding: 'utf-8'
+  };
+
+  const results = await execFileAsync(command, args, options);
+  return results.stdout;
+}
+
+async function execGitCmd(args) {
+  const gitResults = await exec('git', args);
+  return gitResults
+    .trim()
+    .toString()
+    .split('\n');
+}
+
+async function listChangedFiles() {
+  const comparedBranch = process.env.CIRCLECI ? 'origin/master' : 'crafter/2019-11-ui-restructure';
```

Comment:
What's this?

Reply:
It compares local fork clone to the origin remote to determine which files have been changed.